### PR TITLE
Docs/how to use the solution

### PR DESCRIPTION
**Pull request by nelsonsouthworks**
## Purpose
Explain how to operate the Broadcast Development Kit (BDK) through the Backend solution.

### CODE REVIEW

**Review by sw-dcherubini — CHANGES_REQUESTED** [review]
We should add a link to this document in the main README of the repository.

File: docs/how-to-use-the-solution/README.md [hunk 1]
```diff
@@ -0,0 +1,774 @@
...
+  
+## Prerequisites
+- Download and Install [Postman](https://www.postman.com/)
+
+## Authorization
+As endpoints in the solution are securitized in order to make requests to the API it is necessary to generate an access token and use it in the header of the requests, otherwise, the API will respond to requests with HTTP error code 401 (Unauthorized).
+
+In this section we describe how to generate an access token using the app registration used for the SPA or by creating a new one for the client credential flow and use it in each request.
+
+### Client Credential Flow
```
[the remark this answers is not in the document]
Comment (facundoctes) [reply docs/how-to-use-the-solution/README.md:31]: What do you think about linking to the Microsoft documentation where it explains both flows?

[the remark this answers is not in the document]
Comment (sw-dcherubini) [reply docs/how-to-use-the-solution/README.md:31]: @facundoctes Yes, that might be enough.

**Review by sw-dcherubini — CHANGES_REQUESTED** [review]
Looks like we are missing the API endpoint to change / regenerate the RTMP stream key.

File: docs/how-to-use-the-solution/README.md [hunk 2]
```diff
@@ -708,6 +709,23 @@ It is necessary to have a [scheduled meeting](https://support.microsoft.com/en-u
         }
     }
     ```
+### Generate RTMP Stream Key
+1. **Generate RTMP Stream Key:** This endpoint updates the streamKey in the privateContext of the call, used key for RTMP  injections and extractions.
```
Comment (sw-dcherubini) [inline docs/how-to-use-the-solution/README.md]: **nitpick**: We have a double white-space between "RTMP" and "injections". Also lets add a sentence here saying that you only need to use the endpoint if you want to change the RMTP key of the current meeting, and that each meeting has its own RTMP key.

File: docs/how-to-use-the-solution/README.md [hunk 3]
```diff
@@ -438,15 +441,15 @@ It is necessary to have a [scheduled meeting](https://support.microsoft.com/en-u
...
-    "resourceType": {{resourceType}}, 
-    "protocol": {{protocol}}, 
-    "mode": {{mode}},
-    "streamUrl": "{{streamUrl}}",  
-    "streamKey": "{{streamKey}}", 
-    "timeOverlay": {{timeOverlay}},
-    "enableSsl" : {{enableSSl}}
+        "participantId": "{{participantId}}",  
+        "participantGraphId": "{{participantGraphId}}",  
+        "resourceType": "{{resourceType}}",
```
Comment (sw-dcherubini) [inline docs/how-to-use-the-solution/README.md]: **suggestion (blocking)**: The problem with putting these placeholders in double quotes, is that these properties don't expect an string. By putting the double quotes here we are giving the impression the value should be an string.

Instead, lets put an example value and refer to the property name in reference table. E.g.:
```json
{
    "protocol": 1
}
```
| Placeholder | Description |
|-------------|-------------|
| protocol | Use `1` for RTMP extraction protocol |

**Review by sw-dcherubini — CHANGES_REQUESTED** [review]

**Review by sw-dcherubini — APPROVED** [review]

**Pull request merged**